ansible_collections/arista/avd/molecule/cv_deploy/inventory/inventory.yml

```
@@ -8,14 +8,15 @@ SITE1:
       children:
         SITE1_SPINES:
           hosts:
-            s1-spine1:
-            s1-spine2:
+            avd-ci-spine1:
+            avd-ci-spine2:
+            avd-ci-core1:
```

Add additional vars for core1 to allow eAPI-based enabling/disabling of TA streaming (foror testing workspace submission when some of the devices are not actively streaming)
```suggestion
            avd-ci-core1:
              ansible_host: "{{ lookup('env', 'AVD-CI-CORE1-FQDN') }}"
              ansible_user: "{{ lookup('env', 'EOS_EAPI_USERNAME') }}"
              ansible_password: "{{ lookup('env', 'EOS_EAPI_PASSWORD') }}"
              ansible_connection: httpapi
              ansible_httpapi_use_ssl: True
              ansible_httpapi_validate_certs: False
              ansible_network_os: eos
              ansible_httpapi_port: 443
```